tox.ini

```diff
@@ -1,34 +1,39 @@
 [tox]
-envlist = {py27,py34,py35}-{dj1.8,dj1.9,dj1.10},py33-dj1.8,flake8,coverage,docs
+envlist = {py27,py34,py36}-dj1.11,{py34,py36}-dj2.0,{py27,py36}-flake8,coverage,docs
```

Comment:
3.4 but not 3.5?

Reply:
I figured py34 and py36 would cover it, but we could add py35 too if you think it's necessary.